### ISSUE TITLE
convertFromRaw not working for new defined entities

### ISSUE THREAD

**Issue by vnovick** [draft-js-plugins/draft-js-plugins#353]
This is my plugin:

```
const linkPlugin = (config = {}) => {
  return {
    decorators: [{
      strategy: findLinkEntities,
      component: Link
    }]
  }
}

function findLinkEntities(contentBlock, callback) {
  contentBlock.findEntityRanges(
    (character) => {
      const entityKey = character.getEntity();
      return (
        entityKey !== null &&
        Entity.get(entityKey).getType() === 'LINK'
      );
    },
    callback
  );
}

const Link = (props) => {
  const {url} = Entity.get(props.entityKey).getData();
  return (
    <a href={url} style={styles.link}>
      { props.children }
    </a>
  );
};
```

This is how I trigger link:

```
const entityKey = Entity.create('LINK', 'MUTABLE', { url: urlValue });
const newState = RichUtils.toggleLink(editorState, selection, entityKey);
```

All is working fine except when I try to import state from outside the editor. When doing so I do the following:

```
EditorState.createWithContent(
          convertFromRaw(action.config.initialState)
        )
```
All state is passed correctly and editor succeeds in creating imported state except state with Link decorator. It creates state correctly, however strategy is not executed. This happens also when I add another link after initial state was imported. 

Basically it looks like when I import external state, LinkPlugin is not executing at all.

**Comment by vnovick** [comment]
The issue happened because I set editorState to EditorState.createEmpty() on Editor first render. then I changed state to state passed from outside. Basically the fix was not to render editor until I get config from outside. If I don't get it from outside then I render it first time with EditorState.createEmpty.

**Comment by lananelson** [comment]
Interested to see if there is a way to update state "from outside" after component has been rendered with some EditorState. I also could wait for necessary data to load before rendering editor for the first time but ideally I would like to reset the state whenever necessary by passing props from parent component. Is it supported?

**Comment by Zhouzi** [comment]
Sure, you can update the editor's content through DraftJS' api so you'll need to become a bit more familiar with the mechanics. Here's the [documentation](https://facebook.github.io/draft-js/docs/overview.html) but as a quick guide you'll probably need to:

```javascript
// create a new contentState object
const newContentState = ContentState.createFromText('insert your string');

// push the new state into the current editorState's "history"
const newEditorState = EditorState.push(currentEditorState, newContentState);
```
